src/omega_edit/client.ts

```diff
@@ -142,8 +142,7 @@ export function activate(ctx: vscode.ExtensionContext) {
       'extension.data.edit',
       async (
         startServ: boolean = true,
-        port: number | undefined = undefined,
-        fileToEdit: string = ''
```

Comment:
You will need to revert all these changes that deal with `fileToEdit`.

The code involving `fileToEdit` is essential for the API to function properly for the tests. With this change during the tests when the data editor is initially opened it will prompt to select a file which is not the desired behavior during testing. The test still currently passes since it just checks the webview is opened, which it is behind the file picking prompt. In future tests for the data editor we will want to use the content that is inside of the data editor. To do this we need to be able to specify the file we wish to edit so that the editor will contain content and not get stuck on a file prompt.

Reply:
I'm not happy with having this testing cruft in the shipped project.  I've reverted it _temporarily_ and will revisit later.